Microsoft.Toolkit.Uwp.UI.Animations/Extensions/AnimationExtensions.cs

```diff
@@ -89,6 +91,56 @@ public static EasingFunctionBase GetEasingFunction(EasingType easingType)
             }
         }
 
+        /// <summary>
+        /// Generates an <see cref="CompositionEasingFunction"/> to be used with Composition animations
+        /// </summary>
+        /// <param name="easingType">The <see cref="EasingType"/> used to generate the CompositionEasingFunction</param>
+        /// <param name="compositor">The <see cref="Compositor"/></param>
+        /// <returns><see cref="CompositionEasingFunction"/></returns>
+        public static CompositionEasingFunction GetCompositionEasingFunction(EasingType easingType, Compositor compositor)
+        {
+            if (DesignTimeHelpers.IsRunningInLegacyDesignerMode)
+            {
+                return compositor.CreateLinearEasingFunction();
+            }
+
+            return GenerateCompositionEasingFunctionFromEasingType(easingType, compositor);
+        }
+
+        private static CompositionEasingFunction GenerateCompositionEasingFunctionFromEasingType(EasingType easingType, Compositor compositor)
+        {
+            if (easingType == EasingType.Default)
+            {
+                easingType = DefaultEasingType;
+            }
+
+            switch (easingType)
+            {
+                case EasingType.Linear:
+                    return compositor.CreateLinearEasingFunction();
+                case EasingType.Cubic:
+                    return compositor.CreateCubicBezierEasingFunction(new Vector2(0.215f, 0.61f), new Vector2(0.355f, 1f));
```

Comment:
Do we have a reference of where these values came from?

Reply:
ditto

Reply:
Got them from http://easings.net/ and tweaking them with http://cubic-bezier.com/. You think I should add them in the docs somewhere?